specification/edgeorderpartner/resource-manager/Microsoft.EdgeOrderPartner/preview/2020-12-01-preview/edgeorderpartner.json

```diff
@@ -0,0 +1,899 @@
+      "post": {
+        "tags": [
+          "PartnerInventory"
+        ],
+        "summary": "API for linking management resource with inventory",
+        "operationId": "ManageLink",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/locationParameter"
+          },
+          {
+            "$ref": "#/parameters/serialNumberParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "manageLinkRequest",
+            "in": "body",
+            "description": "Links the management resource to the inventory",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ManageLinkRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "\"No content\""
+          },
+          "204": {
+            "description": "\"No content\""
+          },
+          "default": {
+            "description": "Error response describing reason for operation failure.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ManageLink": {
+            "$ref": "./examples/ManageLink.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.EdgeOrderPartner/searchInventories": {
+      "post": {
+        "tags": [
+          "PartnerInventory"
+        ],
+        "summary": "API for Search inventories",
+        "operationId": "SearchInventories",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "searchInventoriesRequest",
+            "in": "body",
+            "description": "Searches inventories with the given filters and returns in the form of a list",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/SearchInventoriesRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/PartnerInventoryList"
```

Comment:
This should be x-ms-pageable with a nextLink as I imagine inventories can/will get quite large (especially with each item having a fairly large body)

Reply:
This is fixed.